contrib/babelfishpg_tsql/src/utils/removeaccent.map

```diff
@@ -0,0 +1,816 @@
+/* TRANSFORMATION_RULE "[[:Latin:][:Nd:]]; Latin-ASCII" */
+
+static const remove_accent_map_pair pltsql_remove_accent_map[811] = 
+{
+    {0x0000c380,0x00000041},
```

Comment:
We should also have automated tests in place to catch any changes made in Unicode entries. By change I mean any new additions, or even change in unaccented form with our rule above.